tests/e2e/scale/test_scale_pvc_pod_multi_project.py

```diff
@@ -0,0 +1,141 @@
+"""
+Testcase to Create 200 PVC+POD scale in one namespace with and without IO
+This will be Scale upto 1500 PVC+POD
+Scale PVC and POD Configuration
+---------------------------------------------------------
+Number of PVC+POD Creation: 1500
+PVC Type: All 4
+POD Type: Nginx basic configuration, no DC POD
+Cluster Capacity to be Utilized: 80%
+Network or IO saturation: 80%
+"""
+import logging
+import pytest
+import random
+
+from tests import helpers
+from ocs_ci.ocs import constants, cluster, machine, node
+from ocs_ci.ocs.resources import pod, pvc
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour, CephHealthException
+from ocs_ci.framework.testlib import scale, E2ETest
+
+log = logging.getLogger(__name__)
+
+
+@scale
+@pytest.mark.parametrize(
+    argnames="start_io",
+    argvalues=[
+        pytest.param(
+            *[False], marks=pytest.mark.polarion_id("OCS-1357")
+        ),
+        pytest.param(
+            *[True], marks=pytest.mark.polarion_id("OCS-1357")
+        )
+    ]
+)
+class TestMultiProjectScalePVCPOD(E2ETest):
+    """
+    Scale the OCS cluster to reach 1500 PVC+POD
+    """
+    @pytest.fixture()
+    def setup_fixture(self, request):
+        def finalizer():
+            self.cleanup()
+
+        request.addfinalizer(finalizer)
+
+    def test_multi_project_scale_pvcs_pods(self, setup_fixture, start_io):
+        """
+        Test case to scale PVC+POD with multi projects and reach expected PVC count
+        """
+
+        # Set PVC count based on number of OCS workers
+        ocs_nodes = node.get_typed_nodes(node_type='worker')
+        pvc_count = {3: 1500, 6: 3000, 9: 4500}
+        scale_count = pvc_count[len(ocs_nodes)]
+
+        # Other variables
+        rbd_sc_obj = helpers.default_storage_class(constants.CEPHBLOCKPOOL)
+        cephfs_sc_obj = helpers.default_storage_class(constants.CEPHFILESYSTEM)
+        pvc_count_each_itr = 5
+        size = f"{random.randrange(5, 105, 5)}Gi"
+        fio_size_param = '2G'
```

Comment:
it would be good to set fio_size and fio_rate according to the number of the osd's and it's size.

Reply:
ok, let me check how to implement it.